src/widgets/splits/SplitInput.cpp

```diff
@@ -604,6 +604,14 @@ void SplitInput::editTextChanged()
     if (text.length() > 0 && getSettings()->showMessageLength)
     {
         labelText = QString::number(text.length());
+        if (text.length() > 500)
```

Comment:
Could this be moved to a const? Makes it easier to find.

If we have another limit of 500 internally (not on Twitch's end) it would be nice to combine those with a common const value.